gui-tk/minsky.tcl

```diff
@@ -223,6 +224,14 @@ if {$preferences(focusFollowsMouse)} {
 }
 proc setCursor {cur} {. configure -cursor $cur; update idletasks}
 
+# Toggle single or multiple equity column mode
+multipleEquities $preferences(multipleEquities)
+if $preferences(multipleEquities) {
+   set multipleEquities 0
+} else {
+   set multipleEquities 1
```

Comment:
Why do you need a separate TCL variable here?

Reply:
I don't know if I'm setting the variable correctly, but using the following does nothing at all, the preference remains single equity column:

if {$preferences(multipleEquities)} {
   set $preferences(multipleEquities) 0
} else {
   set $preferences(multipleEquities) 1
}

Reply:
You are setting a TCL variable correctly, but not using it in any way. You already have a TCL variable anyway, in the form of preferences(multipleEquities), so a multipleEquities TCL variable is redundant.

You need to set the Minksy::multipleEquities attribute for anything to happen.